scripts/install_build_dependencies.sh

```
@@ -11,5 +11,8 @@
 apt-get install -y \
   libdlib-dev:${DEB_HOST_ARCH} libblas-dev:${DEB_HOST_ARCH} libatlas-base-dev:${DEB_HOST_ARCH} liblapack-dev:${DEB_HOST_ARCH} libjpeg62-turbo-dev:${DEB_HOST_ARCH}
 
+# Install gomagic dependencies
+apt-get install -y libmagic-dev:${DEB_HOST_ARCH}
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)